chaoscenter/upgrade-agents/control-plane/versions/v3.9.0/manager.go

```diff
@@ -0,0 +1,71 @@
+package v3_9_0
+
+import (
+	"context"
+	log "github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// VersionManager implements IVersionManger
+type VersionManager struct {
+	Logger   *log.Logger
+	DBClient *mongo.Client
+	Context  *context.Context
+}
+
+// NewVersionManger provides a new instance of a new VersionManager
+func NewVersionManger(logger *log.Logger, dbClient *mongo.Client) *VersionManager {
+	return &VersionManager{Logger: logger, DBClient: dbClient}
+}
+
+// Run executes all the steps required for the Version Manger
+// to upgrade from the previous version to `this` version
+func (vm VersionManager) Run() error {
+	ctx := context.Background()
+	session, err := vm.DBClient.StartSession()
+
+	defer session.EndSession(ctx)
+
+	logVersion := log.Fields{
```

Comment:
Can you please follow this kind of logging here: https://github.com/litmuschaos/litmus/pull/4740/files#diff-c3dcc200129e5965fb7dc2655155eb285cfea424297940f911ffe1b6624b8316R62